shopinvader_api_signin_jwt/routers/signin.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2023 ACSONE SA/NV
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from typing import Annotated
+
+from fastapi import APIRouter, Depends, Response, status
+
+from odoo import api, models
+
+from odoo.addons.base.models.res_partner import Partner
+from odoo.addons.fastapi.dependencies import odoo_env
+from odoo.addons.fastapi_auth_jwt.dependencies import (
+    Payload,
+    auth_jwt_authenticated_payload,
+    auth_jwt_optionally_authenticated_partner,
+)
+
+signin_router = APIRouter(tags=["signin"])
+
+
+@signin_router.post("/signin", status_code=200)
+def signin(
+    # TODO use fastai_auth_jwt.auth_jwt_authenticated_odoo_env when it is fixed
```

Comment:
@qgroulard Can you elaborate on what's the problem? There is no reported issue... 
To move forward you could moves the TODOs into a ROADMAP.rst file.